### DISCUSSION TITLE
Bitrate information

### DISCUSSION THREAD

**Discussion by theelog** [music-assistant/support#2963] [Feature requests and ideas]
I'd like to see bitrate information next to seeing the format (MP3/FLAC). This is also very handy for radio stations which tend to have multiple streams with different bitrates. I personally like to listen to the highest bitrate available, but currently I'm not able to verify.

**Comment by OzGav** [comment]
Try tapping on the format

**Reply by theelog** [reply]
That shows information about bit depth and sample frequency but not bitrate right?

**Reply by OzGav** [reply]
Yes correct. Will see how many others would like this.

**Reply by marcelveldt** [reply]
We dont expose this info right now indeed byt FWIW; we always pick the highest quality stream at all cases

**Reply by OzGav** [reply]
Bit rate is available in the Audio Pipeline display https://beta.music-assistant.io/audiopipeline/

**Reply by woiwoiwoiwoi** [reply]
The Audio Pipeline does not show the bitrate.

**Reply by OzGav** [reply]
Yes it does 
![IMG_0468](https://github.com/user-attachments/assets/6a91219e-27aa-44de-96eb-81048da4fbe0)

**Reply by woiwoiwoiwoi** [reply]
For Spotify yes. For RadioBrowser it does not.
<img width="397" height="448" alt="no-bitrate" src="https://github.com/user-attachments/assets/2be41f21-25cb-4f6b-aa6d-e221af17ed5d" />

**Reply by OzGav** [reply]
bit rate is an optional field which must be added per provider. After review radio browser does supply this via their API so will be added soon.

**Reply by woiwoiwoiwoi** [reply]
Ha, that sounds fantastic, thank you!

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.
